[2022-04-07 19:25:18,537] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: load_to_BigQuery.create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task scheduled__2017-06-01T00:00:00+00:00 [queued]>
[2022-04-07 19:25:18,668] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: load_to_BigQuery.create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task scheduled__2017-06-01T00:00:00+00:00 [queued]>
[2022-04-07 19:25:18,673] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-07 19:25:18,676] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-04-07 19:25:18,678] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-07 19:25:19,704] {taskinstance.py:1270} INFO - Executing <Task(BigQueryInsertJobOperator): create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task> on 2017-06-01 00:00:00+00:00
[2022-04-07 19:25:19,901] {standard_task_runner.py:52} INFO - Started process 4139 to run task
[2022-04-07 19:25:19,995] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'load_to_BigQuery', 'create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task', 'scheduled__2017-06-01T00:00:00+00:00', '--job-id', '12934', '--raw', '--subdir', 'DAGS_FOLDER/load_to_BigQuery_dag.py', '--cfg-path', '/tmp/tmps0wo93he', '--error-file', '/tmp/tmpyls6pxrm']
[2022-04-07 19:25:20,015] {standard_task_runner.py:80} INFO - Job 12934: Subtask create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task
[2022-04-07 19:25:20,568] {logging_mixin.py:109} INFO - Running <TaskInstance: load_to_BigQuery.create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task scheduled__2017-06-01T00:00:00+00:00 [running]> on host 2c54f2187010
[2022-04-07 19:25:21,036] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=load_to_BigQuery
AIRFLOW_CTX_TASK_ID=create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task
AIRFLOW_CTX_EXECUTION_DATE=2017-06-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2017-06-01T00:00:00+00:00
[2022-04-07 19:25:21,044] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-04-07 19:25:21,125] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-04-07 19:25:21,231] {bigquery.py:1554} INFO - Inserting job ***_load_to_BigQuery_create_external_posts_answers_bq_create_posts_answers_stack_overflow_data_partitioned_table_task_2017_06_01T00_00_00_00_00_98eaed14903bf6e6303046eab6979376
[2022-04-07 19:25:22,486] {taskinstance.py:1774} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 2153, in execute
    job = self._submit_job(hook, job_id)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 2119, in _submit_job
    retry=self.result_retry,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/common/hooks/base_google.py", line 439, in inner_wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 1560, in insert_job
    job.result(timeout=timeout, retry=retry)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/job/query.py", line 1498, in result
    do_get_result()
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/job/query.py", line 1488, in do_get_result
    super(QueryJob, self).result(retry=retry, timeout=timeout)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/job/base.py", line 728, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/api_core/future/polling.py", line 135, in result
    raise self._exception
google.api_core.exceptions.BadRequest: 400 Could not cast literal "creation_date" to type TIMESTAMP at [1:109]

Location: us-central1
Job ID: airflow_load_to_BigQuery_create_external_posts_answers_bq_create_posts_answers_stack_overflow_data_partitioned_table_task_2017_06_01T00_00_00_00_00_98eaed14903bf6e6303046eab6979376

[2022-04-07 19:25:22,739] {taskinstance.py:1288} INFO - Marking task as FAILED. dag_id=load_to_BigQuery, task_id=create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task, execution_date=20170601T000000, start_date=20220407T192518, end_date=20220407T192522
[2022-04-07 19:25:22,896] {standard_task_runner.py:98} ERROR - Failed to execute job 12934 for task create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task (400 Could not cast literal "creation_date" to type TIMESTAMP at [1:109]

Location: us-central1
Job ID: airflow_load_to_BigQuery_create_external_posts_answers_bq_create_posts_answers_stack_overflow_data_partitioned_table_task_2017_06_01T00_00_00_00_00_98eaed14903bf6e6303046eab6979376
; 4139)
[2022-04-07 19:25:22,988] {local_task_job.py:154} INFO - Task exited with return code 1
[2022-04-07 19:25:23,168] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-04-07 22:18:14,241] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: load_to_BigQuery.create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task scheduled__2017-06-01T00:00:00+00:00 [queued]>
[2022-04-07 22:18:14,287] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: load_to_BigQuery.create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task scheduled__2017-06-01T00:00:00+00:00 [queued]>
[2022-04-07 22:18:14,288] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-07 22:18:14,288] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-04-07 22:18:14,289] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-07 22:18:14,755] {taskinstance.py:1270} INFO - Executing <Task(BigQueryInsertJobOperator): create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task> on 2017-06-01 00:00:00+00:00
[2022-04-07 22:18:14,787] {standard_task_runner.py:52} INFO - Started process 6311 to run task
[2022-04-07 22:18:14,814] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'load_to_BigQuery', 'create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task', 'scheduled__2017-06-01T00:00:00+00:00', '--job-id', '14173', '--raw', '--subdir', 'DAGS_FOLDER/load_to_BigQuery_dag.py', '--cfg-path', '/tmp/tmpn1i1sk24', '--error-file', '/tmp/tmpsliy1b_x']
[2022-04-07 22:18:14,816] {standard_task_runner.py:80} INFO - Job 14173: Subtask create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task
[2022-04-07 22:18:14,962] {logging_mixin.py:109} INFO - Running <TaskInstance: load_to_BigQuery.create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task scheduled__2017-06-01T00:00:00+00:00 [running]> on host 36107d30f64e
[2022-04-07 22:18:15,098] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=load_to_BigQuery
AIRFLOW_CTX_TASK_ID=create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task
AIRFLOW_CTX_EXECUTION_DATE=2017-06-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2017-06-01T00:00:00+00:00
[2022-04-07 22:18:15,101] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-04-07 22:18:15,118] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-04-07 22:18:15,146] {bigquery.py:1554} INFO - Inserting job ***_load_to_BigQuery_create_external_posts_answers_bq_create_posts_answers_stack_overflow_data_partitioned_table_task_2017_06_01T00_00_00_00_00_7ea769cc58dd870606eb818de391d497
[2022-04-07 22:18:18,978] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=load_to_BigQuery, task_id=create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task, execution_date=20170601T000000, start_date=20220407T221814, end_date=20220407T221818
[2022-04-07 22:18:19,085] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-07 22:18:19,320] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
